content/program.md

```diff
@@ -2,35 +2,209 @@
+day = "2023-11-13"
+from = "13:30"
+to = "15:00"
+talks = [
+    {title="Understanding problems in old MT data using modern methods", authors="Alan G. Jones, Randall Mackie, and Wolfgang Soyer"},
+    {title="Negative transients in central-loop time-domain electromagnetic data: Induced polarization or 3D coupling effect?", authors="Dikun Yang, Ming Cheng, and Qiang Luo"},
+    {title="Impacts of magnetic permeability on electromagnetic data collected in settings with steel-cased wells", authors="Lindsey J. Heagy and Douglas W. Oldenburg"},
+]
+
+[[extra.events]]
+title = "Coffee Break"
+day = "2023-11-13"
+from = "15:00"
+to = "15:30"
+
+[[extra.events]]
+title = "Poster Lightning Talks"
+day = "2023-11-13"
+from = "15:00"
+to = "16:15"
+
+[[extra.events]]
+title = "Poster Reception"
+day = "2023-11-13"
+from = "16:15"
+to = "18:30"
+posters = [
+    {title="3D inversion of semi-airborne magnetometric resistivity data", authors="Zhongchang Chen and Dikun Yang"},
+    {title="Conductivity structure beneath Australia constrained by 3-D inversion of MT Tippers in spherical geometry", authors="Filippo Cicchetti "},
+    {title="3D AMT/MT case histories from Brazil", authors="Patricia de Lugão"},
+    {title="3D minimum-structure inversion for CSEM problems using potentials and unstructured tetrahedral grids", authors="K.B. Kara and C.G. Farquharson"},
+    {title="Targeting epithermal Au-Ag using helicopter VTEM 1D resistivity and 3D magnetic inversion results, and radiometric data at Lawyers Project, North-Central BC, Canada", authors="Karl Kwan, Jean Legault, Jim Greig, Ewan Webster and Mark Hanki"},
+    {title="A parallel adaptive finite-element method for 3-D large-scale controlled-source electromagnetic forward modelling with hierarchical tetrahedral grids", authors="Z. Liu, Z. Ren, H. Uao, J. Tang, X. Lu, and C.G. Farquharson"},
+    {title="Fast 3D magnetotelluric forward modeling using survey decomposition", authors="Lian Liu, Dikun Yang"},
+    {title="Using convolutional neural networks to classify UXO with multicomponent electromagnetic induction data", authors="Jorge Lopez-Alvis, Lindsey J. Heagy, Douglas W. Oldenburg, Stephen Billings and Lin-Ping Song"},
+    {title="2.5D Inversion of frequency-domain Land CSEM data using Broyden's method", authors="Ali Mohand-Saïd"},
+    {title="DESMEX - Evolution of Semi-airborne Controlled-source Electromagnetics for Mineral Exploration and Beyond", authors="The DESMEX Team"},
+    {title="3D Casing-Source Electromagnetic Modeling for CO2 Plumes and Enhanced Geothermal Systems Monitoring", authors="Evan Schankee Um and David L. Alumbaugh"},
+    {title="Petrophysically constrained joint inversion of multi-physical data using the extended alternting direction method of multipliers", authors="Ke Wang, Dikun Yang"},
+    {title="Comparison of MVI with sparse norm susceptibility inversion accounting for demagnetization", authors="John M. Weis, Lindsey J. Heagy, and Douglas W. Oldenburg"},
```

Comment:
can we comment out John's poster for now? He hasn't yet confirmed if he would like to present (will confirm by friday)